app/controllers/signatures_controller.rb

```diff
@@ -15,11 +14,17 @@ def create
 
     rebate_form = RebateForm.find(decoded_token['rebate_form_id'])
 
-    signatures = signature_params.map do |signature|
-      instantiate_signature(signature, rebate_form)
-    end
+    if rebate_form.signatures.exists?
+      render json: {}, status: :forbidden
```

Comment:
403 forbidden usually means an authorization problem, maybe you want 422 Unprocessable instead